ffi/util.lua

```diff
@@ -214,6 +214,23 @@ function util.execute(...)
     end
 end
 
+--- Returns the length of data that can be read immediately without blocking
+function util.availableReadSize(luafile)
```

Comment:
I think "available" says less than "safe". Sure, if you just use "safe" you don't know safe for what without looking it up. But with just "available" the thought that going ahead anyway if it's 0 would have unintended consequences might not cross my mind.

Reply:
For me "safe" is too strong :) there no unintended consequences: the consequences are blocking or non-blocking, nothing unsafe, just blocking :)
util.availableNonBlockingReadSize() ?

Reply:
I'd say getNonBlockingReadSize.

Reply:
OK, this one is fine :)